src/ui/server/controllers/insurance/all-sections/index.ts

```diff
@@ -33,6 +33,11 @@ export const get = (req: Request, res: Response) => {
     application.broker.isUsingBroker,
     application.company.hasDifferentTradingName,
     application.declaration.hasAntiBriberyCodeOfConduct,
+    application.buyer.relationship.exporterIsConnectedWithBuyer,
+    application.buyer.buyerTradingHistory.exporterHasTradedWithBuyer,
+    application.buyer.buyerTradingHistory.outstandingPayments,
+    application.buyer.relationship.exporterHasPreviousCreditInsuranceWithBuyer,
+    application.totalContractValueOverThreshold,
```

Comment:
Update unit test? Not a blocke, we should destructure here now there's a lot of data

Reply:
the test fails when I pass everything and not sure why